config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  kubernetes-sigs/kubebuilder:
+  - name: pull-kubebuilder-test
+    decorate: true
+    always_run: true
+    optional: false
+    path_alias: sigs.k8s.io/kubebuilder
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: golang:1.12
+        command:
+        - ./test.sh
+        env:
+        - name: TRACE
```

Comment:
It is recommended to put everything possible into a script inside the tested repo, because then you can change it in a PR and get direct feedback on that pr.

Reply:
removed env vars and set GO111MODULE=on for installation of two deps in the job. I also removed trace, but we may want to leave that? Doesn't really make sense to put it in the script which is checking for it as well

Reply:
I'd add a dedicated `ci-run-tests.sh` script for that which wraps the existing one